pretix_eth/network/tokens.py

```diff
@@ -42,10 +47,14 @@ def __init__(self):
 
     def _validate_class_variables(self):
         if not (
```

Comment:
In my opinion, using `not all(self.NETWORK_IDENTIFIER, self.NETWORK_VERBOSE_NAME, self.NETWORK_UNISWAP_NAME, self.TOKEN_SYMBOL)` would be a wee bit more elegant without compromising on clarity.

Reply:
oo good idea!